src/main/java/de/dataelementhub/model/handler/element/section/DefinitionHandler.java

```diff
@@ -171,4 +172,20 @@ public static void updateDefinitions(DSLContext ctx, int userId, String urn,
     create(ctx, definitions, elementScopedIdentifier.getElementId(),
         elementScopedIdentifier.getId());
   }
+
+  /**
+   * Check for duplicate languages in definitions.
+   */
+  public static boolean hasDuplicateLanguageDefinitions(List<Definition> definitions) {
```

Comment:
Since it is placed in the DefinitionHandler now, I'd suggest renaming it to simply "hasDuplicateLanguage"

Reply:
done